test/protocol/SellerHandlerTest.js

```diff
@@ -1967,5 +2343,257 @@ describe("SellerHandler", function () {
+
+        await expect(accountHandler.connect(other1).optInToSellerUpdate(seller.id, [SellerFields.Clerk]))
+          .to.emit(accountHandler, "SellerUpdateApplied")
+          .withArgs(
+            seller.id,
+            sellerStruct,
+            pendingSellerUpdateStruct,
+            emptyAuthTokenStruct,
+            emptyAuthTokenStruct,
+            other1.address
+          );
+      });
+
+      it("Should update admin, clerk and operator in a single call ", async function () {
+        seller.clerk = other1.address;
+        seller.admin = other1.address;
+        seller.operator = other1.address;
+        sellerStruct = seller.toStruct();
+
+        await accountHandler.connect(admin).updateSeller(seller, emptyAuthToken);
+
+        await expect(
+          accountHandler
+            .connect(other1)
+            .optInToSellerUpdate(seller.id, [SellerFields.Clerk, SellerFields.Admin, SellerFields.Operator])
+        )
+          .to.emit(accountHandler, "SellerUpdateApplied")
+          .withArgs(
+            seller.id,
+            sellerStruct,
+            pendingSellerUpdateStruct,
+            emptyAuthTokenStruct,
+            emptyAuthTokenStruct,
+            other1.address
+          );
+      });
+
+      it("Should update operator, clerk and auth token in a single call when addresses are the same ", async function () {
+        seller.admin = ethers.constants.AddressZero;
+        seller.operator = authTokenOwner.address;
+        seller.clerk = authTokenOwner.address;
+        sellerStruct = seller.toStruct();
+
+        await accountHandler.connect(admin).updateSeller(seller, authToken);
+
+        await expect(
+          accountHandler
+            .connect(authTokenOwner)
+            .optInToSellerUpdate(seller.id, [SellerFields.Operator, SellerFields.Clerk, SellerFields.AuthToken])
+        )
+          .to.emit(accountHandler, "SellerUpdateApplied")
+          .withArgs(
+            seller.id,
+            sellerStruct,
+            pendingSellerUpdateStruct,
+            authTokenStruct,
+            emptyAuthTokenStruct,
+            authTokenOwner.address
+          );
+      });
+
+      it("New auth token owner should opt-in to update seller", async function () {
+        seller.admin = ethers.constants.AddressZero;
+        sellerStruct = seller.toStruct();
+
+        await accountHandler.connect(admin).updateSeller(seller, authToken);
+
+        await expect(accountHandler.connect(authTokenOwner).optInToSellerUpdate(seller.id, [SellerFields.AuthToken]))
+          .to.emit(accountHandler, "SellerUpdateApplied")
+          .withArgs(
+            seller.id,
+            sellerStruct,
+            pendingSellerUpdateStruct,
+            authTokenStruct,
+            emptyAuthTokenStruct,
+            authTokenOwner.address
+          );
+      });
+
+      context("💔 Revert Reasons", async function () {
+        it("Should revert if there are no pending updates", async function () {
```

Comment:
The use of "Should revert if" in these test cases is inconsistent. It looks like in most places in this file we do not include this text at the begging of the revert reason test name.